docs-src/references/web-ui-server-env-vars.md

```diff
@@ -0,0 +1,249 @@
+---
+id: web-ui-server-env-vars
+title: Temporal Web UI environmental variables
+sidebar_label: Web UI environmental variables
+description: How to set  environmental variables for Temporal Web UI.
+tags:
+  - docker
+  - webui
+  - ui server
+---
+
+Environmental variables are fields that are set to configure a Temporal Cluster.
+This allows the user to configure the Cluster in any environment, such as a Docker container.
+
+Use `docker run` as the image name needed to configure the Web UI environmental variables.
+`docker run` requires at least 1 argument.
+
+`docker run [options] <image> <command> [args]`
+
+The Web UI Server is set up using specific environmental variables.
+These variables need to be passed when running the image.
+
+```
+docker run \
+    -e TEMPORAL_ADDRESS=127.0.0.1:7233 \
+    -e TEMPORAL_UI_PORT=8080 \
+    -e TEMPORAL_AUTH_ENABLED=true \
+    -e TEMPORAL_AUTH_PROVIDER_URL=https://accounts.google.com \
+    -e TEMPORAL_AUTH_CLIENT_ID=xxxxx-xxxx.apps.googleusercontent.com \
+    -e TEMPORAL_AUTH_CLIENT_SECRET=xxxxxxxxxxxxxxx \
+    -e TEMPORAL_AUTH_CALLBACK_URL=https://xxxx.com:8080/auth/sso/callback \
+    -e TEMPORAL_UI_ENABLED=true \
+    -e TEMPORAL_OPENAPI_ENABLED=true \
+    -e TEMPORAL_TLS_CA=../ca.cert \
+    -e TEMPORAL_TLS_CERT=../cluster.pem \
+    -e TEMPORAL_TLS_KEY=../cluster.key \
+    -e TEMPORAL_TLS_ENABLE_HOST_VERIFICATION=true \
+    -e TEMPORAL_TLS_SERVER_NAME=tls-server \
+    temporalio/ui:<tag>
+```
+
+The setup process involves the configuration of several components, including mTLS, authentication, and more.
+
+The environmental variables for the entire configuration template can be found [here](https://github.com/temporalio/ui-server/blob/main/docker/config_template.yaml).
+
+The environmental variables needed to configure the WebUI server environment are explained below.
```

Comment:
"The UI server environment variables are defined in x and described in more detail below... "

This feels like it could be grouped into the intro sentences.